$NetBSD: patch-alire__common.gpr,v 1.3 2025/04/27 10:43:42 dkazankov Exp $

Add NetBSD support, fix switches for release build (avoiding runtime exceptions)

--- alire_common.gpr.orig	2025-03-04 10:02:00.000000000 +0000
+++ alire_common.gpr
@@ -5,6 +5,7 @@
    type Host_OSes is ("linux",
                       "freebsd",
                       "openbsd",
+                      "netbsd",
                       "macos",
                       "windows");
 
@@ -108,7 +109,7 @@
                "-fPIC",
 
                --  Enable lots of extra runtime checks
-               "-gnatVa", "-gnatwa", "-gnato", "-fstack-check", "-gnata",
+               "-gnatwa", "-gnato",
                "-gnatf", "-fPIC")
               & Style_Check_Switches;
 
